docs/adding_validators.md

```diff
@@ -0,0 +1,86 @@
+# Adding Validators
+
+## Overview
+
+New validation logic for addon metadata may require a new validator to be
```

Comment:
"may require" is a bit confusing. Suggestion:

Before creating a new validator, check if the logic should fit an existing validator.

In case you create a new validator, it has to be added to the...

Reply:
Reworded it a bit if you want to check again.